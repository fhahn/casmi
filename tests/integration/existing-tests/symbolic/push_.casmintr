tff(symbolNext, type, sym2: $int).
fof(id0,hypothesis,fa(0,sym2)).%CREATE: a
fof(id1,hypothesis, fpush(sym2,1,sym3)).
fof(id2,hypothesis,fa(1, sym3)).%UPDATE: a
fof(id3,hypothesis,fprogram(0, 0,@undef)).%UPDATE: program 0
fof(final0, hypothesis, fa(1,X) <=>fa(666,X)).
fof(final1, hypothesis, fprogram(0,0,X) <=>fprogram(666,0,X)).
