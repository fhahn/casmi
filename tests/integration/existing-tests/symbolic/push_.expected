tff(symbolNext, type, sym2: $int).
fof(id0,hypothesis,sta(1,sym2)).%CREATE: a 
tff(symbolNext, type, sym3: $int).
fof(id1,hypothesis, fpush(sym2,1,sym3)).
fof(id2,hypothesis,sta(2,sym3)).%UPDATE: a 
fof(final0,hypothesis,sta(0,sym3)).%FINAL: a 
